packages/trader/src/Modules/Contract/Containers/__tests__/contract-replay-widget.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { ChartBottomWidgets, ChartTopWidgets, DigitsWidget, InfoBoxWidget } from '../contract-replay-widget';
+
+jest.mock('../contract-replay-widget.tsx', () => ({
+    ...jest.requireActual('../contract-replay-widget.tsx'),
+    DigitsWidget: () => <div>MockedDigitsWidget</div>,
+    InfoBoxWidget: () => <div>MockedInfoBoxWidget</div>,
```

Comment:
@henry-deriv  please don't mock the widgets which are supposed to be tested, otherewise these tests don't make sense, they will never fail🙏🏻